### DISCUSSION TITLE
Getting an error during final commit for my portfolio website.

### DISCUSSION THREAD

**Discussion by Jayati-Wajire** [microsoft/SATechnicalOnboarding#75] [Questions]
I made the necessary changes and saved them. Then I committed but it was not accepted on github. 
please guide.

![commit error](https://github.com/microsoft/SATechnicalOnboarding/assets/109218479/9e9f6054-395d-477c-a373-5c7fa594f978)
![commit error1](https://github.com/microsoft/SATechnicalOnboarding/assets/109218479/67e2352d-9716-474a-8c15-fa905cb23cde)

**Comment by AnthonyByansi** [comment]
Hey @Jayati-Wajire It looks like you made the commits but you didn't push your changes, correct me if I'm wrong.

**Reply by Jayati-Wajire** [reply]
Yes, but I am unable to push it. It says there is one problem in this file, which I mentioned above.
